edwin/client/static/js/components/Timeline.js

```diff
@@ -24,22 +27,34 @@ import Icon from './Icon.js';
  */
 export default class Timeline extends ControllerComponent {
   get stores() {
-    return [BugStore, UserStore];
+    return [BugStore, UserStore, IndicatorStore];
   }
 
   loadData() {
     let teamSlug = this.props.params.team;
     let bugQuery = {comment_tag: `edwin-${teamSlug}`};
 
+    IndicatorActions.updateState(LoadingStates.USER_RESTORE);
```

Comment:
This would probably also give more information so we could distinguish between `LOAD_BUGS` and `LOAD_BUG_COMMENTS` (the second is the one that probably takes the most time).